org.alloytools.kodkod.core/.settings/org.eclipse.jdt.core.prefs

```diff
@@ -21,8 +21,11 @@ org.eclipse.jdt.core.codeComplete.staticFinalFieldSuffixes=
 org.eclipse.jdt.core.compiler.annotation.inheritNullAnnotations=disabled
 org.eclipse.jdt.core.compiler.annotation.missingNonNullByDefaultAnnotation=ignore
 org.eclipse.jdt.core.compiler.annotation.nonnull=org.eclipse.jdt.annotation.NonNull
+org.eclipse.jdt.core.compiler.annotation.nonnull.secondary=
```

Comment:
Please don't gratuitously change the preferences in this PR.

If you want to suggest better preferences, please open a new PR.